docs/accounts/account/import_view_only_account.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Can we move these changes to another PR for improving documentation of view-only accounts?